app/controllers/short_url_requests_controller.rb

```diff
@@ -49,6 +51,21 @@ def reject
     redirect_to short_url_requests_path
   end
 
+  def destroy
+    router.delete_route("#{@short_url_request.from_path}")
```

Comment:
This shouldn't be hitting the router directly.  For creating routes, this uses the publishing-api, so removing them should also use it.

There's no mechanism to delete entries from the content-store via the publishing-api, this is deliberate.  Instead, items should be replaced with a [gone item](https://github.com/alphagov/content-store/blob/master/doc/gone_item.md) so that requests return a 410.